wqflask/tests/wqflask/show_trait/test_export_trait_data.py

```diff
@@ -3,10 +3,72 @@
 from wqflask.show_trait.export_trait_data import dict_to_sorted_list
 from wqflask.show_trait.export_trait_data import cmp_samples
 from wqflask.show_trait.export_trait_data import export_sample_table
+from wqflask.show_trait.export_trait_data import get_export_metadata
+
+
+class MockGeneral(object):
+    def __init__(self, obj):
+        for key, value in obj.items():
+            setattr(self, key, value)
+
+
+class MockChild(object):
+    pass
```

Comment:
**issue:** Could you please give these better names?

Use more descriptive names that make the intent of the class. It makes reading your code easier.